descriptor.json

```diff
@@ -0,0 +1,29 @@
+{
+        "package": {
+        "name": "vaani.iot", 
+        "repo": "openhab-distro",
+        "subject": "vaani/vaani.iot",
+        "desc": "Vaani OpenHab submodule",
+        "website_url": "www.vaani.iot",
+        "vcs_url": "https://github.com/mozilla/openhab-distro.git",
+        "github_use_tag_release_notes": true,
+        "github_release_notes_file": "README.md",
+        "labels": ["vaani", "openhab", "mozilla"],
+        "public_download_numbers": false,
+        "public_stats": false,
```

Comment:
The deployment still fails because of [a parsing error](https://s3.amazonaws.com/archive.travis-ci.org/jobs/116764125/log.txt).

If the parser is super strict, this line is problematic.